static/inline/js/inline.js

```diff
@@ -222,7 +222,6 @@
                         settings: {
                             arrows: false,
                             centerPadding: generalOptions.product_holder_slide_center_padding,
-                            centerMode : true,
```

Comment:
When there's only 1 product no need to run the slider as it brakes the layout

Reply:
done